app/validation.py

```diff
@@ -1,81 +1,47 @@
 import json
 import re
+import os
 
 from flask import abort
-from jsonschema import validate, ValidationError
+from jsonschema import ValidationError, FormatChecker
 from jsonschema.validators import validator_for
 
 MINIMUM_SERVICE_ID_LENGTH = 10
 MAXIMUM_SERVICE_ID_LENGTH = 20
 
-with open("json_schemas/g4-services-schema.json") as g4_schema:
-    G4_SCHEMA = json.load(g4_schema)
-    G4_VALIDATOR = validator_for(G4_SCHEMA)
-    G4_VALIDATOR.check_schema(G4_SCHEMA)
-    G4_VALIDATOR = G4_VALIDATOR(G4_SCHEMA)
-
-with open("json_schemas/g5-services-schema.json") as g5_schema:
-    G5_SCHEMA = json.load(g5_schema)
-    G5_VALIDATOR = validator_for(G5_SCHEMA)
-    G5_VALIDATOR.check_schema(G5_SCHEMA)
-    G5_VALIDATOR = G5_VALIDATOR(G5_SCHEMA)
-
-with open("json_schemas/g6-scs-schema.json") as g6_scs_schema:
-    G6_SCS_SCHEMA = json.load(g6_scs_schema)
-    G6_SCS_VALIDATOR = validator_for(G6_SCS_SCHEMA)
-    G6_SCS_VALIDATOR.check_schema(G6_SCS_SCHEMA)
-    G6_SCS_VALIDATOR = G6_SCS_VALIDATOR(G6_SCS_SCHEMA)
-
-with open("json_schemas/g6-saas-schema.json") as g6_saas_schema:
-    G6_SAAS_SCHEMA = json.load(g6_saas_schema)
-    G6_SAAS_VALIDATOR = validator_for(G6_SAAS_SCHEMA)
-    G6_SAAS_VALIDATOR.check_schema(G6_SAAS_SCHEMA)
-    G6_SAAS_VALIDATOR = G6_SAAS_VALIDATOR(G6_SAAS_SCHEMA)
-
-with open("json_schemas/g6-iaas-schema.json") as g6_iaas_schema:
-    G6_IAAS_SCHEMA = json.load(g6_iaas_schema)
-    G6_IAAS_VALIDATOR = validator_for(G6_IAAS_SCHEMA)
-    G6_IAAS_VALIDATOR.check_schema(G6_IAAS_SCHEMA)
-    G6_IAAS_VALIDATOR = G6_IAAS_VALIDATOR(G6_IAAS_SCHEMA)
-
-with open("json_schemas/g6-paas-schema.json") as g6_paas_schema:
-    G6_PAAS_SCHEMA = json.load(g6_paas_schema)
-    G6_PAAS_VALIDATOR = validator_for(G6_PAAS_SCHEMA)
-    G6_PAAS_VALIDATOR.check_schema(G6_PAAS_SCHEMA)
-    G6_PAAS_VALIDATOR = G6_PAAS_VALIDATOR(G6_PAAS_SCHEMA)
-
-with open("json_schemas/update-details.json") as update_json:
-    UPDATER_SCHEMA = json.load(update_json)
-    UPDATER_VALIDATOR = validator_for(UPDATER_SCHEMA)
-    UPDATER_VALIDATOR.check_schema(UPDATER_SCHEMA)
-    UPDATER_VALIDATOR = UPDATER_VALIDATOR(UPDATER_SCHEMA)
-
-with open("json_schemas/users.json") as json_file6:
-    USERS_SCHEMA = json.load(json_file6)
-    USERS_VALIDATOR = validator_for(USERS_SCHEMA)
-    USERS_VALIDATOR.check_schema(USERS_SCHEMA)
-    USERS_VALIDATOR = USERS_VALIDATOR(USERS_SCHEMA)
-
-with open("json_schemas/auth_users.json") as json_file7:
-    AUTH_USERS_SCHEMA = json.load(json_file7)
-    AUTH_USERS_VALIDATOR = validator_for(AUTH_USERS_SCHEMA)
-    AUTH_USERS_VALIDATOR.check_schema(AUTH_USERS_SCHEMA)
-    AUTH_USERS_VALIDATOR = AUTH_USERS_VALIDATOR(AUTH_USERS_SCHEMA)
+JSON_SCHEMAS_PATH = './json_schemas'
+FORMAT_CHECKER = FormatChecker()
+
+
+def load_schemas(schemas_path):
+    loaded_schemas = {}
+    for schema_file in os.listdir(schemas_path):
```

Comment:
Talked with @robyoung and decided to replace the autoloading with an explicit list of schema names. That way we'll  know that the required schema files exist on start.